src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/GlobalSearchService.kt

```diff
@@ -31,7 +30,6 @@ class GlobalSearchService(
     private val log: Logger = LoggerFactory.getLogger(this::class.java)
   }
 
-  @PreAuthorize("hasRole('GLOBAL_SEARCH')")
```

Comment:
Moved up to the endpoint to allow us to sneak around this authorization when calling from within the deployment.